src/picongpu/include/particles/ionization/byField/BSIHydrogenLike/AlgorithmBSIHydrogenLike.hpp

```diff
@@ -64,13 +65,16 @@ namespace ionization
         operator()( const BType bField, const EType eField, ParticleType& parentIon )
         {
 
-            const float_X protonNumber = GetAtomicNumbers<ParticleType>::type::numberOfProtons;
-            float_X chargeState = attribute::getChargeState(parentIon);
-
-            uint32_t cs = math::float2int_rd(chargeState);
+            const float_X protonNumber  = GetAtomicNumbers<ParticleType>::type::numberOfProtons;
+            float_X chargeState         = attribute::getChargeState(parentIon);
+            uint32_t cs                 = math::float2int_rd(chargeState);
+            /* ionization potential in atomic units */
+            const float_X iEnergy       = GetIonizationEnergies<ParticleType>::type()[cs];
+            /* critical field strength in atomic units */
+            float_X critField           = math::pow(iEnergy,float_X(2.)) / protonNumber;
```

Comment:
please use `math::pow(iEnergy,2)` or `iEnergy*iEnergy`.

The current used pow function with two floats is very slow.